### PR TITLE
Adding alpha.12 to the list of spelling exclusions

### PR DESCRIPTION
**Pull request by mirichmo**

### CODE REVIEW

**Review by lzybkr — COMMENTED** [review]
Maybe we should preemptively add a bunch of these?

File: .spelling [hunk 1]
```diff
@@ -77,6 +77,7 @@ init
 labeled
  - demos/Azure/README.md
 alpha.11
```
Comment (daxian-dbw) [inline .spelling]: `alpha.11` shouldn't be needed now, otherwise it means we left some docs not updated 😄

Comment (mirichmo) [inline .spelling]: `> get-childitem .\.spelling | select-string "alpha."
.spelling:79:alpha.11
.spelling:80:alpha.12
.spelling:93:alpha.8
.spelling:201:alpha.9
.spelling:258:alpha.7`
There are a bunch of others as well

Comment (mirichmo) [inline .spelling]: The others are included in the docs and aren't really worth updating. I removed alpha.11 since it was not in the docs.

Comment (daxian-dbw) [inline .spelling]: Thanks!

**Pull request merged**